src/Build/Logging/BinaryLogger/Postprocessing/IBuildEventArgsReaderNotifications.cs

```diff
@@ -6,7 +6,10 @@ namespace Microsoft.Build.Logging
     /// <summary>
     /// An interface for notifications from BuildEventArgsReader
     /// </summary>
-    public interface IBuildEventArgsReaderNotifications : IBuildEventStringsReader
+    public interface IBuildEventArgsReaderNotifications :
+        IBuildEventStringsReader,
+        IBuildFileReader,
+        IBinlogReaderErrors
     {
         /* For future use */
```

Comment:
Is this comment still valid? It seems to be in use.

Reply:
This was meant for the actual members of the interface.

But that comment for sure can be confusing while having questionable to no value - so I'll ditch it